core/modules/layout/layout.module

```diff
@@ -366,9 +368,9 @@ function layout_page_title(Layout $layout) {
   // here is intentional. See backdrop_get_title().
   $path = $layout->getPath();
   if ($path) {
-    return t('Edit "!layout" at !path', array('!layout' => $layout->title, '!path' => $path));
+    return t('!layout', array('!layout' => $layout->title));
   }
-  return t('Edit "!layout"', array('!layout' => $layout->title));
+  return t('!layout', array('!layout' => $layout->title));
```

Comment:
@docwilmot because this is now returning the same thing either way, is there any reason to still get the path and have this if check here? Otherwise this looks very nice.